packages/recoil/core/Recoil_RecoilEnv.js

```diff
@@ -12,14 +12,61 @@
 'use strict';
 
 const err = require('recoil-shared/util/Recoil_err');
+const gkx = require('recoil-shared/util/Recoil_gkx');
 
 export type RecoilEnv = {
   RECOIL_DUPLICATE_ATOM_KEY_CHECKING_ENABLED: boolean,
+  RECOIL_GKS_ENABLED_UNSTABLE: string[],
 };
 
-const env: RecoilEnv = {
-  RECOIL_DUPLICATE_ATOM_KEY_CHECKING_ENABLED: true,
-};
+const env: RecoilEnv = new Proxy(
```

Comment:
I couldn't think of a better way to implement this than using a proxy (or moving `RecoilEnv` into the shared package and having `Recoil_gkx` reference it, but that seemed like a bigger refactor), but if there is one, I'd be happy to take a suggestion to improve it!

Reply:
This does seem a bit overly complex.  Moving to shared would be totally fine.  Actually, that would be safer to ensure the GK module is updated before any actual GK checks are made, depending on the order the modules are run.